controller/cmd/public-api/main.go

```diff
@@ -25,7 +25,7 @@ func main() {
 	kubeConfigPath := flag.String("kubeconfig", "", "path to kube config")
 	prometheusURL := flag.String("prometheus-url", "http://127.0.0.1:9090", "prometheus url")
 	metricsAddr := flag.String("metrics-addr", ":9995", "address to serve scrapable metrics on")
-	proxyAPIAddr := flag.String("proxy-api-addr", "127.0.0.1:8086", "address of proxy-api service")
+	proxyAPIAddr := flag.String("destination-addr", "127.0.0.1:8086", "address of destination service")
```

Comment:
Can we also rename this variable to `destinationAPIAddr`? And then for completeness later in this file, `proxyAPIConn` to `destinationAPIConn`?